### DISCUSSION TITLE
How can I remove the lag ?

### DISCUSSION THREAD

**Discussion by sasankaweera123** [sasankaweera123/Face_Recognition_Project_0001#6] [Q&A] [answered]
- when the image is captured from the camera there is a lag when showing the name, how to remove this lag?

**Comment by PramudithaPothuwila** [comment, marked as answer]
Try to implement the capture process and recognition in two threads

**Reply by PramudithaPothuwila** [reply]
Use IPython because CPython has the global interpreter lock
